tests/transforms/test_qcut.py

```diff
@@ -729,3 +747,46 @@ def test_communication_graph_persistence(self):
 
         assert communication_graph_0.nodes == communication_graph_1.nodes
         assert communication_graph_0.edges == communication_graph_1.edges
+
+
+class TestGraphToTape:
+    def test_graph_to_tape(self):
+        """
+        Tests that a directed multigraph is correctly converted to a tape
+        """
+
+        g = qcut.tape_to_graph(multi_cut_tape)
```

Comment:
We also need a circuit whose fragments have mid-circuit measurements, acting as a nice test of the wire swapping functionality.